pkg/driver/driver_alicloud.go

```diff
@@ -90,7 +92,9 @@ func (c *AlicloudDriver) Create() (string, string, error) {
 	response, err := client.RunInstances(request)
 	if err != nil {
 		return "", "", err
+		metrics.ApiFailedRequestCount.With(prometheus.Labels{"provider": "alicloud", "service": "ecs"}).Inc()
```

Comment:
I think you will have to move this statement above the return as mentioned by @dkistner. The lint checks are failing.

Reply:
Oh, cool catch. Fixing.